src/Assets/Api.php

```diff
@@ -0,0 +1,146 @@
+<?php
+/**
+ * Contains asset api methods
+ *
+ * @package WooCommerce/Blocks
+ */
+
+namespace Automattic\WooCommerce\Blocks\Assets;
+
+use Automattic\WooCommerce\Blocks\Domain\Package;
+
+/**
+ * The Api class provides an interface to various asset registration helpers.
+ *
+ * @since $VID:$
+ */
+class Api {
+
+	/**
+	 * Reference to the Package instance
+	 *
+	 * @var Package
+	 */
+	private $package;
+
+	/**
+	 * Constructor for class
+	 *
+	 * @param Package $package An instance of Package.
+	 */
+	public function __construct( Package $package ) {
+		$this->package = $package;
+	}
+	/**
```

Comment:
Nitpicking here, but I would add a blank line here...
```suggestion

	/**
```